### PR TITLE
Remove redirect_uri check for request_uri before building request object

### PR DESCRIPTION
**Pull request by thiloshon**
Resolves https://github.com/wso2/product-is/issues/9401

Description

PAR request expects `redirect_uri` to be omitted from the authorization call [1] as it is already added to the request object in PAR endpoint. Also in OAuth, redirect_uri is an optional parameter [2]. This PR removes the check for `redirect_uri` when authorization call comes with `request_uri`

[1] https://tools.ietf.org/html/draft-ietf-oauth-par-03#section-4
[2] https://tools.ietf.org/html/rfc6749#section-4.1.1

### CODE REVIEW

**Comment by emswbandara** [comment]
@thiloshon there are some conflicting files. can you resolve them?

File: components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/CodeTokenResponseValidator.java [hunk 1]
```diff
@@ -58,6 +64,11 @@ private static boolean isContainOIDCScope(String scope) {
     @Override
     public void validateRequiredParameters(HttpServletRequest request) throws OAuthProblemException {
 
+        if (!StringUtils.isBlank(request.getParameter(REQUEST_URI))) {
```
Comment (emswbandara) [inline components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/CodeTokenResponseValidator.java:67]: shall we use isNotBlank here?

**Comment by emswbandara** [comment]
@thiloshon can you check why travis is failing? did you the build with unit tests locally?

**Comment by thiloshon** [comment]
Closing as the PR is stale now

**Pull request closed without merging**